README.md

```diff
@@ -138,6 +140,36 @@ function debug(plugins) {
 }
 ```
 
+##### `encapsulateStyles`
+
+* Type: `Boolean`
+* Default: `false`
+
+Resets CSS properties to their default values to effectively allow a component
+to opt out of CSS inheritance and be encapsulated from the rest of the
+application. There are two types of CSS properties that affect components, inherited (e.g. `font-size,`
+`color`) and non-inherited (e.g. `margin`, `background`). This option works so that:
+
+* Root elements (e.g. `.Component`) have both inherited and non-inherited
+  properties reset to default values.
+* Descendants (e.g. `.Component-item`) only have non-inherited properties reset
+  as this allows properties set on the root element to be applied to
+  descendants.
+
+This means that components are isolated from styles outside the component root
+element but should an inheritable property such as `font-size` be applied on the
+component root element it will be inherited by the component descendants as
+normal. The same rules apply to nested components.
+
+To achieve this [postcss-autoreset](https://github.com/maximkoretskiy/postcss-autoreset)
+is used with the SUIT preset and a custom set of CSS properties to reset. Only
+selectors conforming to the SUIT naming conventions are effected.
+
+###### Examples
+
+* [Component encapsulated from body styles](http://codepen.io/simonsmith/pen/82e55b2d58c9aa8dff6f950df68f147c)
+* [Nested components](http://codepen.io/simonsmith/pen/f95d0dba64553d2860ecd8a5fba1eb87)
```

Comment:
Maybe add a comment around the generated reset to explain that `preprocessor` added em.
It could be useful to style a link and show how it is reset in a nested component.